src/SFA.DAS.PR.Application/Requests/Commands/CreateAddAccountRequest/CreateAddAccountRequestCommand.cs

```diff
@@ -8,12 +8,13 @@ namespace SFA.DAS.PR.Application.Requests.Commands.CreateAddAccountRequest;
 public class CreateAddAccountRequestCommand : IRequest<ValidatedResponse<CreateAddAccountRequestCommandResult>>, IUkprnEntity, IOperationsEntity, IAccountLegalEntityIdEntity
 {
     public required long? Ukprn { get; set; }
-    
+
     public required string RequestedBy { get; set; }
 
     public required long AccountLegalEntityId { get; set; }
 
     public string? EmployerContactEmail { get; set; }
+    public string? Paye { get; set; }
```

Comment:
This is now required for AddAccount so we should add validation for it

Reply:
agree